indy_node/server/domain_req_handler.py

```diff
@@ -252,6 +423,32 @@ def _validate_revoc_reg_def(self, req: Request):
         assert revoc_def_tag
         assert revoc_def_type
 
+    def get_current_revoc_entry_and_revoc_def(self, author_did, revoc_reg_def_id, req_id):
+        assert author_did
+        assert revoc_reg_def_id
+        # TODO Need to be "isCommited"==True?
```

Comment:
Since we call this method as part of dynamic validation (that is for new txns added to uncommitted list), we must get the uncommitted values to take into account all txns in the batch